extensions/amp-sidebar/0.2/test/test-amp-sidebar.js

```diff
@@ -202,6 +202,16 @@ describes.realWin(
         });
       });
 
+      it('should create styleable mask element in DOM', () => {
+        return getAmpSidebar({'stubHistory': true}).then((sidebarElement) => {
```

Comment:
nit:
```suggestion
      it('should create styleable mask element in DOM', async () => {
        const sideBarElement = await getAmpSidebar({'stubHistory': true});
```

Reply:
Good call. I refactored all of the tests in these files with the async/await pattern.